eve/main.yml

```diff
@@ -180,9 +197,15 @@ stages:
             OS_USERNAME: "%(secret:scality_cloud_username)s"
             OS_PASSWORD: "%(secret:scality_cloud_password)s"
             OS_TENANT_NAME: "%(secret:scality_cloud_tenant_name)s"
-            TF_VAR_worker_uuid: "%(prop:worker_uuid)s"
+            # FIXME: this makes hostnames too long
+            # TF_VAR_worker_uuid: "%(prop:worker_uuid)s"
             TF_VAR_nodes_count: "2"
           haltOnFailure: true
+      - SetPropertyFromCommand:
+          name: Save spawned bootstrap IP
+          workdir: build/eve/workers/openstack-multiple-nodes/terraform/
```

Comment:
minor: in this file we use `"` around this path, so you may want to remove them since we obviously don't need them

Reply:
Tracked [here](https://github.com/scality/metalk8s/issues/781#issuecomment-476565028).